packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -215,17 +225,22 @@ Future<bool> get isFlutterGAEnabled async {
 /// Request Flutter tool stored property value clientID (GA enabled) stored in
 /// the file '~\.flutter'.
 ///
-/// Return as a String or null, null implies Flutter Tool has never been run.
+/// Return as a String, empty string implies Flutter Tool has never been run.
 Future<String> flutterGAClientID() async {
+  // Default empty string, Flutter tool never run.
   String clientId = '';
 
   if (isDevToolsServerAvailable) {
-    final resp = await _request(server.apiGetFlutterGAClientId);
-    if (resp?.status == HttpStatus.ok) {
-      // Return value 'null' implies Flutter tool has never been run return null.
-      clientId = json.decode(resp.responseText);
-    } else {
-      _logWarning(resp, server.apiGetFlutterGAClientId);
+    // Test if Flutter is enabled (or if Flutter Tool ever run if not enabled is
+    // false, we don't want to be the first to create a ~/.flutter file.
```

Comment:
nit: end parenthesis in comment

Reply:
Was changing comment  - now its done. Good catch.

Reply:
Good catch I fixed another comment not this one. Thanks.